protocol/contracts/v2/strategies/RamosStrategy.sol

```diff
@@ -51,12 +54,20 @@ contract RamosStrategy  is AbstractStrategy, IRamosProtocolTokenVault {
         templeToken.mint(recipient, amount);
     }
 
+    function borrowQuoteToken(uint256 amount, address recipient) external {
+        quoteToken.safeTransfer(recipient, amount);
```

Comment:
For the v2 strategy, it should be borrowing the funds from the TRV and sending to the recipient.

`treasuryReservesVault.borrow()`